packages/web3-providers/src/Trending/helpers.ts

```diff
@@ -10,7 +10,7 @@
 
 export function getCommunityLink(links: string[]): TrendingAPI.CommunityUrls {
     return links.map((x) => {
-        if (x.includes('twitter')) return { type: 'twitter', link: x }
+        if (x.includes('twitter') || x.includes('x.com')) return { type: 'twitter', link: x }
```

Comment:
@guanbinrui @swkatmask you must use this

```ts
import { isDomainOrSubdomainOf } from '@masknet/shared-base'

if (isDomainOrSubdomainOf(x, 'x.com')) return { type: 'twitter', link: x }
```

otherwise, v2ex.com also passes this test.